canonical/vanilla-framework
# Remove medium breakpoint

## Done

Removed `$breakpoint-medium` from Vanilla and replace it with `--large` or `--small` breakpoints where relevant.
Updated documentation and examples of affected components.
Added 

Fixes #4083 

## QA

Review examples of relevant patterns, make sure they work as expected on different screen sizes:

https://vanilla-framework-4105.demos.haus/docs/examples/layouts/application/default
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/forms/form-inline
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/forms/form-stacked
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/lists/lists-split
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/lists/lists-stepped-detailed
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/matrix
https://vanilla-framework-4105.demos.haus/docs/examples/patterns/tables/table-mobile-card
https://vanilla-framework-4105.demos.haus/docs/examples/templates/responsive

Review updated docs to see if they make sense and don't mention medium breakpoint:

https://vanilla-framework-4105.demos.haus/docs/layouts/application
https://vanilla-framework-4105.demos.haus/docs/patterns/grid
https://vanilla-framework-4105.demos.haus/docs/patterns/lists
https://vanilla-framework-4105.demos.haus/docs/settings/breakpoint-settings
https://vanilla-framework-4105.demos.haus/docs/upgrade-guide-v3
